src/Plugin/Field/FieldFormatter/FieldLinkLabelTextOnly.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
This is not really needed if we use https://www.drupal.org/project/link_title_formatter

Reply:
Oh, that is a good find. Let me check that out.

Reply:
This will have to be paired with the https://www.drupal.org/project/display_field_copy module as well.

Reply:
oh yay!

Reply:
Those two contrib might be a nicer experience as you won't get the two extra fields for every link by default and have to create them one by one.

Reply:
Replaced with contrib modules and re-exported config into stanford_profile.